__mocks__/passport.js

```diff
@@ -33,13 +33,19 @@ const user = {
 };
 
 passport.aUser = user;
+passport.global = {};
+passport.setGlobal = (key, value) => {
+  passport.global[key] = value;
+};
+passport.clearGlobal = (key) => { delete passport.global[key]; };
 
 function authenticate(strategy, options) {
   'use strict';
 
   return (req, res, next) => {
     req.user = passport.aUser;
     req.isAuthenticated = () => true; // Skip calling jwtStrategy, auth the request straight
+    // req.body = passport.global[req.originalUrl];
```

Comment:
probably shouldn't include this commented out code in commit

Reply:
sure missed that